src/helpers/html.js

```diff
@@ -1,6 +1,16 @@
+// Component is the base class for a react component.
+// PropTypes provides type checking for props in dev mode.
 import React, {Component, PropTypes} from 'react';
+
+// ReactDOMServer is the API for rendering React components from
```

Comment:
To elaborate on this: `ReactDOMServer is the API for rendering React components into an HTML string. This is done on the server.`